linux_files/setup

```diff
@@ -89,44 +76,89 @@ function byemessage {
     exit 0
 }
 
+function mainmenu{
+
+SELECTION=$(whiptail --title "WLinux Setup Main Menu" --menu "Enter Selects" 0 0 0 --cancel-button Quit --ok-button Select \
+  "1 EXPLORER" "Enable right-click on folders in Windows Explorer to open them in WLinux" \
+  "2 SHELLS" "Install and configure default shells from bash, zsh, csh, and fish" \
+  "3 EDITORS" "Install text editors neovim, emacs, or Visual Studio Code" \
+  "4 PYTHON" "Install Python 3.7 and download and install latest PIP" \
+  "5 NODEJS"  "Install Node.js" \
+  "6 GO" "Install the latest Go" \
+  "7 POWERSHELL and AZURE" "Install PowerShell for Linux and Azure CLI Tools" \
+  "8 BASIC GUI TOOLS" "Install the basic libraries needed for most GUI apps and optionally set HiDPI" \
+  "q QUIT" "Quit This Menu Program"  3>&1 1>&2 2>&3)
+  
+  RET=$?
+  if [ $RET -eq 1 ]; then
+    exit 0
+  elif [ $RET -eq 0 ]; then
+    case "$SELECTION" in
+      1\ *) explorerintegration
+            mainmenu ;;
+      2\ *) installandsetshell
+            fzfinstall
+            mainmenu ;;
+      3\ *) neoviminstall
+            emacsinstall
+            codeinstall
+            mainmenu ;;
+      4\ *) pythoninstall
+            mainmenu ;;
+      5\ *) nodeinstall
+            mainmenu ;;
+      6\ *) goinstall
+            mainmenu ;;
+      7\ *) powershellinstall
+            azurecliinstall
+            mainmenu ;;
+      8\ *) basicguiinstall
+            hidpiconfig
+            mainmenu ;;
+      q\ *) echo "Exiting menu..."
```

Comment:
`;;` is missing at the end

Reply:
Got it.

Is n in your apt repo match supporting testing?